app/main.py

```diff
@@ -1,3 +1,55 @@
+from __future__ import annotations
+import math
+
+
 class Vector:
-    # write your code here
-    pass
+    def __init__(self, x: int, y: int) -> None:
+        self.x = round(x, 2)
+        self.y = round(y, 2)
+
+    def __add__(self, vector_2: Vector) -> Vector:
```

Comment:
W definicji takich metod używamy słówka `other` (taka konwencja)
```suggestion
    def __add__(self, other: Vector) -> Vector:
```